src/transformers/utils/hub.py

```diff
@@ -75,17 +80,25 @@ def is_offline_mode():
 
 
 torch_cache_home = os.getenv("TORCH_HOME", os.path.join(os.getenv("XDG_CACHE_HOME", "~/.cache"), "torch"))
+default_cache_path = constants.default_cache_path
```

Comment:
From the change below (`default_cache_path` --> `constants.HUGGINGFACE_HUB_CACHE`)`, I don't know well why here we don't use `default_cache_path = constants.HUGGINGFACE_HUB_CACHE`.

Reply:
`default_cache_path` is `os.path.join(HF_HOME, "hub")` while `HUGGINGFACE_HUB_CACHE` is potentially set by the user to a different location.

To be honest I'd be in favor of removing it completely as it is used nowhere but I kept it for backward compatibility (`from transformers.utils import default_cache_path` is currently exposed on `main` branch)